crates/ruma-macros/src/api/version.rs

```diff
@@ -4,12 +4,19 @@ use proc_macro2::TokenStream;
 use quote::{format_ident, quote, ToTokens};
 use syn::{parse::Parse, Error, LitFloat};
 
-#[derive(Clone)]
+#[derive(Clone, Debug, PartialEq)]
 pub struct MatrixVersionLiteral {
     pub(crate) major: NonZeroU8,
     pub(crate) minor: u8,
 }
 
+const ONE: NonZeroU8 = unsafe { NonZeroU8::new_unchecked(1) };
```

Comment:
Is there no way to circumvent this `unsafe` code block?

Reply:
There is but the `unsafe` is trivially sound and the safe way to write it is longer and uglier.